crypto/des/des_local.h

```diff
@@ -109,6 +109,19 @@
                                         : "cc");        \
                            ret;                         \
                         })
+#  elif defined(__riscv_zbb) || defined(__riscv_zbkb)
+#   if __riscv_xlen == 64
+#    define ROTATE(x, n) ({ int ret;           \
+                       asm ("roriw %0, %1, %2" \
```

Comment:
Should this use `u32 ret`?

Reply:
Changed. I used the `register unsigned int` way in x86 code above because `u32` is not defined here.